2_adventures/001/jonkernpa/04_day/README.md

```diff
@@ -0,0 +1,11 @@
+# Adventure Day 4
+
+Just what is an adventure?
+
+How lame... spent an hour cleaning up my lousy mess of git rebase hell.
+
+     git remote add upstream /url/to/original/repo
+     git fetch upstream
+     git checkout master
+     git reset --hard upstream/master
+     git push origin master --force
```

Comment:
Thanks for the encouraging words, @kotp 
So, periodically, I should get the latest in Pat's original repo via:

    git merge upstream/master

And then make my next topic branch:

    git checkout -b jk-next-adventure-nn

Does that sound right?